src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Common/ProjectPackageCollections.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    /// <summary>
+    /// Struct containing installed and transitive package collections for a project
+    /// </summary>
+    public struct ProjectPackageCollections
```

Comment:
`PackageCollections` contains packages for 1 _or more_ projects.  Given that project information has been stripped away by the time this object is created, how about renaming this more appropriately to `InstalledAndTransitivePackageCollections` and update the XML doc comment.

Reply:
Fixed